types/src/system/auction/mod.rs

```diff
@@ -286,12 +297,12 @@ pub trait Auction:
         Ok(new_delegation_amount)
     }
 
-    /// Removes an amount of motes (or the entry altogether, if the remaining amount is 0) from
-    /// the entry in delegators and calls unbond in the Mint contract to create a new unbonding
-    /// purse.
+    /// Removes specified amount of motes (or the value from the collection altogether, if the
+    /// remaining amount is 0) from the entry in delegators map for given validator and creates a
+    /// new unbonding request to the queue.
     ///
-    /// The arguments are the delegator’s key, the validator key and quantity of motes and
-    /// returns a tuple of the unbonding purse along with the remaining bid amount.
+    /// The arguments are the delegator's key, the validator's key, and the remaining stake and
+    /// return the remaining bid amount.
```

Comment:
I'm a little confused by "return the remaining bid amount".

I think it'd be good to have this as a separate sentence, but that aside, can we try and be fairly consistent in our description of the return types for all the methods being documented in this PR please?